diff --git a/panels/applications/meson.build b/panels/applications/meson.build
index 8fdffd374..81508984e 100644
--- a/panels/applications/meson.build
+++ b/panels/applications/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
         input : desktop + '.in',
        output : desktop,
        po_dir : po_dir,
-      install : true,
+      install : false,
   install_dir : control_center_desktopdir
 )
 
diff --git a/panels/background/meson.build b/panels/background/meson.build
index 87c615bac..03904f467 100644
--- a/panels/background/meson.build
+++ b/panels/background/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/color/meson.build b/panels/color/meson.build
index 764e0f0fb..934b59784 100644
--- a/panels/color/meson.build
+++ b/panels/color/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/display/meson.build b/panels/display/meson.build
index 39d0ee813..e7b692f95 100644
--- a/panels/display/meson.build
+++ b/panels/display/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/keyboard/meson.build b/panels/keyboard/meson.build
index 34e5f620b..5920f9f90 100644
--- a/panels/keyboard/meson.build
+++ b/panels/keyboard/meson.build
@@ -7,7 +7,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/mouse/meson.build b/panels/mouse/meson.build
index d96d893cf..5a6561491 100644
--- a/panels/mouse/meson.build
+++ b/panels/mouse/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/multitasking/meson.build b/panels/multitasking/meson.build
index 4bc9cab0e..831a005e0 100644
--- a/panels/multitasking/meson.build
+++ b/panels/multitasking/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/notifications/meson.build b/panels/notifications/meson.build
index 787758d39..ec096f198 100644
--- a/panels/notifications/meson.build
+++ b/panels/notifications/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/online-accounts/meson.build b/panels/online-accounts/meson.build
index 67300d010..92fb72530 100644
--- a/panels/online-accounts/meson.build
+++ b/panels/online-accounts/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/privacy/meson.build b/panels/privacy/meson.build
index 4360b10cb..41b7143e3 100644
--- a/panels/privacy/meson.build
+++ b/panels/privacy/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/search/meson.build b/panels/search/meson.build
index 984c12099..f43e581bc 100644
--- a/panels/search/meson.build
+++ b/panels/search/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/sharing/meson.build b/panels/sharing/meson.build
index 34805e0d5..7cf05f7ce 100644
--- a/panels/sharing/meson.build
+++ b/panels/sharing/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/system/about/meson.build b/panels/system/about/meson.build
index a987d20cb..4bc89d931 100644
--- a/panels/system/about/meson.build
+++ b/panels/system/about/meson.build
@@ -4,7 +4,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/system/datetime/meson.build b/panels/system/datetime/meson.build
index 1babf242d..0e35046a2 100644
--- a/panels/system/datetime/meson.build
+++ b/panels/system/datetime/meson.build
@@ -4,7 +4,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/system/meson.build b/panels/system/meson.build
index dc83af881..0d27ef216 100644
--- a/panels/system/meson.build
+++ b/panels/system/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/system/region/meson.build b/panels/system/region/meson.build
index 44fa40ec7..db15d9bb5 100644
--- a/panels/system/region/meson.build
+++ b/panels/system/region/meson.build
@@ -4,7 +4,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/system/users/meson.build b/panels/system/users/meson.build
index 97f89f570..f764b306d 100644
--- a/panels/system/users/meson.build
+++ b/panels/system/users/meson.build
@@ -4,7 +4,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/universal-access/meson.build b/panels/universal-access/meson.build
index b06c13a3b..7e184e061 100644
--- a/panels/universal-access/meson.build
+++ b/panels/universal-access/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/wacom/meson.build b/panels/wacom/meson.build
index a72e31261..f999d6ae8 100644
--- a/panels/wacom/meson.build
+++ b/panels/wacom/meson.build
@@ -51,7 +51,7 @@ i18n.merge_file(
         input : desktop + '.in',
        output : desktop,
        po_dir : po_dir,
-      install : true,
+      install : false,
   install_dir : control_center_desktopdir
 )
 
diff --git a/shell/cc-panel-list.c b/shell/cc-panel-list.c
index 2b59587b1..ed38df374 100644
--- a/shell/cc-panel-list.c
+++ b/shell/cc-panel-list.c
@@ -381,32 +381,8 @@ static const gchar * const panel_order[] = {
   "display",
   "sound",
   "power",
-  "multitasking",
-  "background",
 
-  "separator",
-
-  "applications",
-  "notifications",
-  "search",
-  "online-accounts",
-  "sharing",
-  "wellbeing",
-
-  "separator",
-
-  "mouse",
-  "keyboard",
-  "color",
   "printers",
-  "wacom",
-
-  "separator",
-
-  "universal-access",
-  "privacy",
-  "system",
-  "reset-settings",
 };
 
 static guint
diff --git a/shell/cc-panel-loader.c b/shell/cc-panel-loader.c
index 28a1671d9..69813207e 100644
--- a/shell/cc-panel-loader.c
+++ b/shell/cc-panel-loader.c
@@ -86,34 +86,16 @@ extern void cc_wwan_panel_static_init_func (void);
 
 static CcPanelLoaderVtable default_panels[] =
 {
-  PANEL_TYPE("applications",     cc_applications_panel_get_type,         NULL),
-  PANEL_TYPE("background",       cc_background_panel_get_type,           NULL),
 #ifdef BUILD_BLUETOOTH
   PANEL_TYPE("bluetooth",        cc_bluetooth_panel_get_type,            NULL),
 #endif
-  PANEL_TYPE("color",            cc_color_panel_get_type,                NULL),
-  PANEL_TYPE("display",          cc_display_panel_get_type,              NULL),
-  PANEL_TYPE("keyboard",         cc_keyboard_panel_get_type,             NULL),
-  PANEL_TYPE("mouse",            cc_mouse_panel_get_type,                NULL),
-  PANEL_TYPE("multitasking",     cc_multitasking_panel_get_type,         NULL),
 #ifdef BUILD_NETWORK
   PANEL_TYPE("network",          cc_network_panel_get_type,              NULL),
   PANEL_TYPE("wifi",             cc_wifi_panel_get_type,                 cc_wifi_panel_static_init_func),
 #endif
-  PANEL_TYPE("notifications",    cc_notifications_panel_get_type,        NULL),
-  PANEL_TYPE("online-accounts",  cc_online_accounts_panel_get_type,      NULL),
   PANEL_TYPE("power",            cc_power_panel_get_type,                NULL),
   PANEL_TYPE("printers",         cc_printers_panel_get_type,             NULL),
-  PANEL_TYPE("privacy",          cc_privacy_panel_get_type,              NULL),
-  PANEL_TYPE("search",           cc_search_panel_get_type,               NULL),
-  PANEL_TYPE("sharing",          cc_sharing_panel_get_type,              cc_sharing_panel_static_init_func),
   PANEL_TYPE("sound",            cc_sound_panel_get_type,                NULL),
-  PANEL_TYPE("system",           cc_system_panel_get_type,               NULL),
-  PANEL_TYPE("universal-access", cc_ua_panel_get_type,                   NULL),
-#ifdef BUILD_WACOM
-  PANEL_TYPE("wacom",            cc_wacom_panel_get_type,                cc_wacom_panel_static_init_func),
-#endif
-  PANEL_TYPE("wellbeing",        cc_wellbeing_panel_get_type,            NULL),
 #ifdef BUILD_WWAN
   PANEL_TYPE("wwan",             cc_wwan_panel_get_type,                 cc_wwan_panel_static_init_func),
 #endif
