diff --git a/panels/applications/meson.build b/panels/applications/meson.build
index b6ff4a0e7..fcc4e626f 100644
--- a/panels/applications/meson.build
+++ b/panels/applications/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
         input : desktop + '.in',
        output : desktop,
        po_dir : po_dir,
-      install : true,
+      install : false,
   install_dir : control_center_desktopdir
 )
 
diff --git a/panels/background/meson.build b/panels/background/meson.build
index 8f9cbecdf..8d974162c 100644
--- a/panels/background/meson.build
+++ b/panels/background/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/color/meson.build b/panels/color/meson.build
index 52198ec59..c3132ccea 100644
--- a/panels/color/meson.build
+++ b/panels/color/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/display/meson.build b/panels/display/meson.build
index c3dd03212..9d7ce2abd 100644
--- a/panels/display/meson.build
+++ b/panels/display/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/keyboard/meson.build b/panels/keyboard/meson.build
index cd1b54e8e..910fff4d5 100644
--- a/panels/keyboard/meson.build
+++ b/panels/keyboard/meson.build
@@ -7,7 +7,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/mouse/meson.build b/panels/mouse/meson.build
index 2cc5048c7..cf6bd3898 100644
--- a/panels/mouse/meson.build
+++ b/panels/mouse/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/multitasking/meson.build b/panels/multitasking/meson.build
index 66680ef32..eb30d1bb2 100644
--- a/panels/multitasking/meson.build
+++ b/panels/multitasking/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/notifications/meson.build b/panels/notifications/meson.build
index c42decc33..8d1275e25 100644
--- a/panels/notifications/meson.build
+++ b/panels/notifications/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/online-accounts/meson.build b/panels/online-accounts/meson.build
index 679566069..23943bf1b 100644
--- a/panels/online-accounts/meson.build
+++ b/panels/online-accounts/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/privacy/meson.build b/panels/privacy/meson.build
index 3afd326be..b9ddb2df2 100644
--- a/panels/privacy/meson.build
+++ b/panels/privacy/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/search/meson.build b/panels/search/meson.build
index df1498e5b..b09e03878 100644
--- a/panels/search/meson.build
+++ b/panels/search/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/sharing/meson.build b/panels/sharing/meson.build
index bab2588f0..dd6859eca 100644
--- a/panels/sharing/meson.build
+++ b/panels/sharing/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/system/about/meson.build b/panels/system/about/meson.build
index 3317e401b..5eb63fe38 100644
--- a/panels/system/about/meson.build
+++ b/panels/system/about/meson.build
@@ -4,7 +4,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/system/datetime/meson.build b/panels/system/datetime/meson.build
index cd940851e..e3666be47 100644
--- a/panels/system/datetime/meson.build
+++ b/panels/system/datetime/meson.build
@@ -4,7 +4,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/system/meson.build b/panels/system/meson.build
index a9ac171e2..58955ad68 100644
--- a/panels/system/meson.build
+++ b/panels/system/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/system/region/meson.build b/panels/system/region/meson.build
index d0ec6ccf6..4690a52f6 100644
--- a/panels/system/region/meson.build
+++ b/panels/system/region/meson.build
@@ -4,6 +4,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
+
diff --git a/panels/system/users/meson.build b/panels/system/users/meson.build
index 5fda2edda..66198a4d5 100644
--- a/panels/system/users/meson.build
+++ b/panels/system/users/meson.build
@@ -4,7 +4,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/universal-access/meson.build b/panels/universal-access/meson.build
index 121f05915..efdf0802b 100644
--- a/panels/universal-access/meson.build
+++ b/panels/universal-access/meson.build
@@ -6,7 +6,7 @@ i18n.merge_file(
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
-  install: true,
+  install: false,
   install_dir: control_center_desktopdir
 )
 
diff --git a/panels/wacom/meson.build b/panels/wacom/meson.build
index 3424eb70a..1dd6324f1 100644
--- a/panels/wacom/meson.build
+++ b/panels/wacom/meson.build
@@ -27,7 +27,7 @@ i18n.merge_file(
         input : desktop + '.in',
        output : desktop,
        po_dir : po_dir,
-      install : true,
+      install : false,
   install_dir : control_center_desktopdir
 )
 
diff --git a/shell/cc-panel-list.c b/shell/cc-panel-list.c
index c3390546e..80bf3ea92 100644
--- a/shell/cc-panel-list.c
+++ b/shell/cc-panel-list.c
@@ -375,32 +375,8 @@ static const gchar * const panel_order[] = {
   "display",
   "sound",
   "power",
-  "multitasking",
-  "background",
 
-  "separator",
-
-  "applications",
-  "notifications",
-  "search",
-  "online-accounts",
-  "sharing",
-  "wellbeing",
-
-  "separator",
-
-  "mouse",
-  "keyboard",
-  "color",
   "printers",
-  "wacom",
-
-  "separator",
-
-  "universal-access",
-  "privacy",
-  "system",
-  "reset-settings",
 };
 
 static guint
diff --git a/shell/cc-panel-loader.c b/shell/cc-panel-loader.c
index 28a1671d9..69813207e 100644
--- a/shell/cc-panel-loader.c
+++ b/shell/cc-panel-loader.c
@@ -86,34 +86,16 @@ extern void cc_wwan_panel_static_init_func (void);
 
 static CcPanelLoaderVtable default_panels[] =
 {
-  PANEL_TYPE("applications",     cc_applications_panel_get_type,         NULL),
-  PANEL_TYPE("background",       cc_background_panel_get_type,           NULL),
 #ifdef BUILD_BLUETOOTH
   PANEL_TYPE("bluetooth",        cc_bluetooth_panel_get_type,            NULL),
 #endif
-  PANEL_TYPE("color",            cc_color_panel_get_type,                NULL),
-  PANEL_TYPE("display",          cc_display_panel_get_type,              NULL),
-  PANEL_TYPE("keyboard",         cc_keyboard_panel_get_type,             NULL),
-  PANEL_TYPE("mouse",            cc_mouse_panel_get_type,                NULL),
-  PANEL_TYPE("multitasking",     cc_multitasking_panel_get_type,         NULL),
 #ifdef BUILD_NETWORK
   PANEL_TYPE("network",          cc_network_panel_get_type,              NULL),
   PANEL_TYPE("wifi",             cc_wifi_panel_get_type,                 cc_wifi_panel_static_init_func),
 #endif
-  PANEL_TYPE("notifications",    cc_notifications_panel_get_type,        NULL),
-  PANEL_TYPE("online-accounts",  cc_online_accounts_panel_get_type,      NULL),
   PANEL_TYPE("power",            cc_power_panel_get_type,                NULL),
   PANEL_TYPE("printers",         cc_printers_panel_get_type,             NULL),
-  PANEL_TYPE("privacy",          cc_privacy_panel_get_type,              NULL),
-  PANEL_TYPE("search",           cc_search_panel_get_type,               NULL),
-  PANEL_TYPE("sharing",          cc_sharing_panel_get_type,              cc_sharing_panel_static_init_func),
   PANEL_TYPE("sound",            cc_sound_panel_get_type,                NULL),
-  PANEL_TYPE("system",           cc_system_panel_get_type,               NULL),
-  PANEL_TYPE("universal-access", cc_ua_panel_get_type,                   NULL),
-#ifdef BUILD_WACOM
-  PANEL_TYPE("wacom",            cc_wacom_panel_get_type,                cc_wacom_panel_static_init_func),
-#endif
-  PANEL_TYPE("wellbeing",        cc_wellbeing_panel_get_type,            NULL),
 #ifdef BUILD_WWAN
   PANEL_TYPE("wwan",             cc_wwan_panel_get_type,                 cc_wwan_panel_static_init_func),
 #endif
